.github/workflows/integration.yml

```diff
@@ -82,7 +80,7 @@ jobs:
       - name: Setup Archery
         run: pip install -e dev/archery[docker]
       - name: Execute Docker Build
-        run: archery docker run conda-integration
+        run: archery docker run -e ARCHERY_INTEGRATION_WITH_RUST=1 conda-integration
```

Comment:
@jorgecarleitao @andygrove @alamb you'll need to update the integration workflow in arrow-rs once we merge this

Reply:
fielded https://github.com/apache/arrow-rs/pull/253